apps/src/applab/applab.js

```diff
@@ -666,6 +666,9 @@ Applab.init = function(config) {
   studioApp.runButtonClick = this.runButtonClick.bind(this);
 
   Applab.channelId = config.channel;
+  if (config.level.testAssetPathPrefix) {
+    Applab.assetPathPrefix = config.level.testAssetPathPrefix;
+  }
```

Comment:
We do get coverate of both the default asset path prefix and the onload behavior, by configuring the test cases differently. In the test "remixed images have correct urls", we leave the image path prefix alone, and verify that the URLs are updated correctly. In the test "remixed images are sized properly", we do some URL validation, and also test that the image sizes are correct after the images load.